zapisy/apps/enrollment/records/models/opening_times.py

```diff
@@ -191,24 +199,42 @@ def is_enrollment_open(cls, course: CourseInstance, time: datetime):
 
     @classmethod
     @transaction.atomic
-    def populate_opening_times(cls, semester: Semester):
-        """Computes opening times (bonuses) for students that cast votes.
-
-        Voting for a course results in a quicker enrollment. The function will
-        throw a DatabaseError if operation is unsuccessful.
+    def populate_opening_times(cls, semester: Semester, *,
+                               students: Iterable[Student] = None, groups: Iterable[Group] = None):
+        """Computes opening times for selected students that cast votes.
+
+        Voting for a course results in a quicker enrollment.
+
+        Args:
+            semester: Semester for which we calculate opening times.
+            students: Students for whom we calculate opening times. If None,
+            calculation will be carried out for all active students.
+            groups: Groups for which we calculate opening times. If None,
+            calculation will be carried out for all groups in the specified
+            semester.
```

Comment:
Mój automat by to wciął jednak 😁 
```suggestion
            semester: Semester for which we calculate opening times.
            students: Students for whom we calculate opening times. If None,
                calculation will be carried out for all active students.
            groups: Groups for which we calculate opening times. If None,
                calculation will be carried out for all groups in the specified
                semester.
```

Reply:
Zrobiłem to Rewrapem, używąc Alt+q